tests/playwright-test/ui-mode-trace.spec.ts

```diff
@@ -554,7 +555,7 @@ test('skipped steps should have an indicator', async ({ runUITest }) => {
   await expect(skippedMarker).toHaveAccessibleName('skipped');
 });
 
-test('should show copy prompt button in errors tab', async ({ runUITest }) => {
+test.fixme('should show copy prompt button in errors tab', async ({ runUITest }) => {
```

Comment:
Why is this one not working? Update expectations instead?

Reply:
because I haven't re-established the rendering yet. once the TODO in the PR is resolved, this test should just work again.